lunamark/reader/markdown.lua

```diff
@@ -150,21 +150,30 @@ parsers.indented_blocks = function(bl)
 end
 
 -- Attributes list as in Pandoc {#id .class .class-other key=value key2="value 2"}
-parsers.identifier  = parsers.letter
-                      * (parsers.alphanumeric + S("_-"))^0
-parsers.attrvalue   = (parsers.dquote * C((parsers.alphanumeric + S("._- "))^1) * parsers.dquote)
-                      + C((parsers.alphanumeric + S("._-"))^1)
-
-parsers.attrpair    = Cg(C((parsers.identifier)^1)
+--   Note: Pandoc has the same identifier definitions, but its interpretation
+--   of a letter (and by extension an alphanumeric value) extends to alphabetic
+--   *Unicode* characters.
+parsers.attrid      = parsers.letter
+                      * (parsers.alphanumeric + S("_-:."))^0
+
+parsers.attrvalue   = (parsers.dquote
+                      * C((parsers.anyescaped - parsers.dquote)^0)
+                      * parsers.dquote)
+                    + (parsers.squote
+                      * C((parsers.anyescaped - parsers.squote)^0)
+                      * parsers.squote)
+                    + C((parsers.anyescaped - parsers.dquote - parsers.space - P("}"))^1)
+
+parsers.attrpair    = Cg(C((parsers.attrid)^1)
```

Comment:
Why `(parsers.attrid)^1`? Doesn't `parsers.attrid` already handle repetition?

Reply:
No idea, it was already on the previous implementation, here I just changed `parsers.identifier` to `parsers.attrid`...

Admittedly, the _previous_ code was also my contribution, but that was more than one year ago, I can't recollect the details of my thinking.

Sure, the change here doesn't make things worse, then.
Could it be made better and is the `^1` needed (if I am getting the question right), I don't know, and I'm afraid I won't have the bandwidth for investigating and testing.

Reply:
Let's merge the following suggestion and the tests should tell us whether it's ok or not:

```suggestion
parsers.attrpair    = Cg(C(parsers.attrid)
```